### ISSUE TITLE
Creating new parts

### ISSUE THREAD

**Issue by albfan** [fritzing/fritzing-app#3361]
Using 0.9.3

After run fritzing and don't find core parts look into fritzing executables

```
$ which Fritzing
/usr/bin/Fritzing
$ which fritzing
/usr/bin/fritzing
```
Looking inside Fritzing

```
$ cat $(which Fritzing)
#!/bin/sh

cd /usr/share/fritzing/parts
exec /usr/bin/fritzing
```

Is it possible to add a config path in preferences? so there's only a run script, and you can configure where to look for parts

Don't know exactly how config files are stored on fritzing but a simple ini file would work

```
$ cat ./config/fritzing
[parts]
path = /usr/share/fritzing/parts
```

Let me know if you find this interesting, to provide a PR (as I said maybe the store engine is different, please add some hints) and if it can be expanded to more desirable config options

**Comment by KjellMorgenstern** [comment]
On linux systems, Fritzing uses $HOME/.config/Fritzing/Fritzing.conf to store settings.
It would be interesting to add a option to the CLI to set a custom parts folder, for example to allow global with a writeable, shared parts folder.

I don't think it make sense to remove the launcher scripts, if that is what you meant?

**Comment by albfan** [comment]
`$HOME/.config/Fritzing/Fritzing.conf` looks nice.

The subject for issue doesn't explain what I want, but only one of the needs, let me rephrase it

**Comment by albfan** [comment]
I see https://fritzing.org/learning/tutorials/creating-custom-parts/

but I'm not sure where those new parts ends. 

the whole idea is to load parts base on `Fritzing.conf` and avoid to `cd` to locate them.

Maybe that could allow to run Fritzing with a specific set of parts easily.

**Comment by KjellMorgenstern** [comment]
The new part is stored in  "My Parts". 
In the Inspector, you can click the file names, and the folder should open, no manual directory search required.

![image](https://user-images.githubusercontent.com/6494025/68923188-f7ec8200-077d-11ea-9517-3587b8e6c998.png)

The parts will be created in $HOME/Documents/Fritzing/parts

**Comment by albfan** [comment]
Nice then. The script cd step shouldn't be needed but is not a problematic thing at all.

Thanks for taking time to explain this, really appreciated.

**Comment by KjellMorgenstern** [comment]
Taking a note to highlight the clickable items in inspector on mouseover events.
